app/population/middleware/set-results.population-table.js

```diff
@@ -1,23 +1,32 @@
-const presenters = require('../../../common/presenters')
-const locationsFreeSpacesService = require('../../../common/services/locations-free-spaces')
+const {
+  decorateAsDateTable,
+} = require('../../../common/presenters/date-table/date-table-decorator')
+const {
+  locationsToPopulationTable,
+} = require('../../../common/presenters/date-table/locations-to-population-table-component')
+const {
+  getPrisonFreeSpaces,
+} = require('../../../common/services/locations-free-spaces')
```

Comment:
This should be changed to use the service on the when #1128 has been merged. cc @Thource